### PR TITLE
Fix #134 - Use renamed soap api (javax.* -> jakarta.*)

### LINKED ISSUE #134: Use renamed soap api (javax.* -> jakarta.*)

### PR DESCRIPTION
**Pull request by tvallin**
Signed-off-by: tvallin <thibault.vallin@oracle.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -165,17 +165,17 @@
...
                 <groupId>jakarta.xml.bind</groupId>
                 <artifactId>jakarta.xml.bind-api</artifactId>
-                <version>2.3.2</version>
+                <version>3.0.0-RC1</version>
             </dependency>
             <dependency>
                 <groupId>jakarta.xml.soap</groupId>
                 <artifactId>jakarta.xml.soap-api</artifactId>
-                <version>1.4.1</version>
+                <version>2.0.1-SNAPSHOT</version>
```
Comment (smillidge) [inline pom.xml]: Is there no RC for this?

File: pom.xml [hunk 2]
```diff
@@ -185,7 +185,7 @@
             <dependency>
                 <groupId>org.jvnet.staxex</groupId>
                 <artifactId>stax-ex</artifactId>
-                <version>1.8.2</version>
+                <version>1.9.0-SNAPSHOT</version>
```
Comment (smillidge) [inline pom.xml]: Is there any version better than a snapshot?

File: saaj-ri/pom.xml [hunk 3]
```diff
@@ -323,17 +323,17 @@
...
                 <groupId>jakarta.xml.bind</groupId>
                 <artifactId>jakarta.xml.bind-api</artifactId>
-                <version>2.3.3</version>
+                <version>3.0.0-RC1</version>
             </dependency>
             <dependency>
                 <groupId>jakarta.xml.soap</groupId>
                 <artifactId>jakarta.xml.soap-api</artifactId>
-                <version>1.4.2</version>
+                <version>2.0.1-SNAPSHOT</version>
```
Comment (smillidge) [inline saaj-ri/pom.xml]: See comments above RE snapshots

File: saaj-samples/saaj-simple/src/main/java/saaj/sender/SendingServlet.java [hunk 4]
```diff
@@ -20,24 +20,24 @@
...
+import jakarta.xml.soap.SOAPBodyElement;
+import jakarta.xml.soap.SOAPConnection;
+import jakarta.xml.soap.SOAPConnectionFactory;
+import jakarta.xml.soap.SOAPEnvelope;
+import jakarta.xml.soap.SOAPException;
+import jakarta.xml.soap.SOAPHeader;
+import jakarta.xml.soap.SOAPMessage;
+import jakarta.xml.soap.SOAPPart;
 import javax.servlet.ServletConfig;
 import javax.servlet.ServletContext;
```
Comment (smillidge) [inline saaj-samples/saaj-simple/src/main/java/saaj/sender/SendingServlet.java]: shouldn't this be the new jakarta servlet api?

File: pom.xml (hunk 1 above)
Comment (tvallin) [reply pom.xml]: Yes there is, I updated the version in my new commit.

File: pom.xml (hunk 2 above)
Comment (tvallin) [reply pom.xml]: Unfortunately, there is not at the moment.

**Comment by lukasj** [comment]
can you rebase this on master and integrate latest dependencies? Those dependencies are:
```
[INFO]   com.sun.activation:jakarta.activation ............. 1.2.2 -> 2.0.0-RC2
[INFO]   com.sun.xml.fastinfoset:FastInfoset ............... 1.2.18 -> 2.0.0-M1
[INFO]   jakarta.activation:jakarta.activation-api ......... 1.2.2 -> 2.0.0-RC2
[INFO]   jakarta.xml.bind:jakarta.xml.bind-api ............. 2.3.3 -> 3.0.0-RC2
[INFO]   jakarta.xml.soap:jakarta.xml.soap-api ............. 1.4.2 -> 2.0.0-RC2
[INFO]   org.jvnet.staxex:stax-ex ........................... 1.8.3 -> 2.0.0-M1
```

**Comment by lukasj** [comment]
you may want to run `mvn org.glassfish.copyright:glassfish-copyright-maven-plugin:repair -Dcopyright.update=true` to fix all copyright headers in one go....

**Comment by lukasj** [comment]
good, I'll take a look at the failing test

**Comment by lukasj** [comment]
@smillidge is there anything what cannot be done in a later PR against master?

**Comment by lukasj** [comment]
tests are failing mostly because service provider files were not renamed...

**Pull request merged**